src/common/entity/binary_sensor_icon.ts

```diff
@@ -7,6 +7,8 @@ export const binarySensorIcon = (state: HassEntity) => {
   switch (state.attributes.device_class) {
     case "battery":
       return activated ? "hass:battery" : "hass:battery-outline";
+    case "battery_charging":
+      return activated ? "hass:battery" : "hass:battery-charging";
```

Comment:
These should be switched right?
Now it shows `hass:battery-charging` when not charging

Reply:
No, I think it's correct as it is.
```python
  const activated = state.state && state.state === "off";
```

So if state is off, show battery else show battery-charging

Reply:
wtf... yeah :+1: